stock_orderpoint_manual_procurement/wizards/make_procurement_orderpoint_item.py

```diff
@@ -0,0 +1,39 @@
+from odoo import api, fields, models
```

Comment:
remove this new file, it's not needed because the code is already in the other file.

Reply:
Thanks for your comment. We will work on this.

Reply:
Good day @MiquelRForgeFlow , this PR has been updated. Please help to review and let us know if anything need to be updated.